lms/djangoapps/instructor/tests/test_email.py

```diff
@@ -10,7 +10,7 @@
 from opaque_keys.edx.keys import CourseKey
 
 from common.djangoapps.student.tests.factories import AdminFactory
-from lms.djangoapps.bulk_email.api import is_bulk_email_enabled_for_course, is_bulk_email_feature_enabled
+from lms.djangoapps.bulk_email.models_api import is_bulk_email_enabled_for_course, is_bulk_email_feature_enabled
```

Comment:
I'm not sure how this worked in the past but ended up as test errors in my changes. These functions were imported into the bulk_email app's `api.py` file but were never part of this file (and exist in the `models_api.py` file).